code_examples/sdk_examples/src/core_features/getting_started/05_fetch_endpoint_data.ts

```diff
@@ -10,6 +10,6 @@ export async function main(
   } = await axios.get<Kilt.KiltPublishedCredentialCollectionV1>(
     endpoints[0].serviceEndpoint[0]
   )
-
+  console.log(`Credentials: ${credential}`)
```

Comment:
```suggestion
  console.log(`Credentials: ${JSON.stringify(credential, null, 2)}`)
```

Reply:
Inside 06 we also log out a credential. Do we want to do it twice? I don't mind since they are learning.

What if we fetch all the available credentials and display them or we could take the claim details in the last log when we state  it is valid and has been attested.

Reply:
@Dudleyneedham I think for now, we just remove the log in 06 and just keep the output of the attester value and that it was valid, it seems enough and emphasis the flow more.